### PR TITLE
Feat/Create contact

### PR DESCRIPTION
**Pull request by soerenstein**
This pull request is linked to issue #13 

Please review the following files of my first user story: 
**CreateContact.js
CreateContact.test.js
CreateContact.stories.js**

### CODE REVIEW

File: src/components/CreateContact/CreateContact.js [hunk 1]
```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/macro'
+
+export default function CreateContact( onSubmit ) {
```
Comment (mpagels) [inline src/components/CreateContact/CreateContact.js:4]: where is onSubmit comming from? Do you need it?

File: src/App.js [hunk 2]
```diff
@@ -1,26 +1,21 @@
...
-    </div>
-  );
+    <Router>
+      <Switch>
+        <Route path="/create">
+          <CreateContact />
+        </Route>    
+      </Switch>
+    </Router>
+  )
```
Comment (mpagels) [inline src/App.js:18]: Nice to have, but do you really need Router, to render one component?

File: src/components/CreateContact/CreateContact.js [hunk 3]
```diff
@@ -0,0 +1,119 @@
...
+                    <input 
+                    name="phone"
+                    />  
+                </LabelStyled>
+                <LabelStyled>
+                    E-Mail
+                    <input 
+                    name="mail"
+                    />  
+                </LabelStyled>
```
Comment (mpagels) [inline src/components/CreateContact/CreateContact.js:51]: Have a look here: [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) you can pass some types for input element. type=email. it's better for accessibilty for blind people for example.

File: src/components/CreateContact/CreateContact.js (hunk 1 above)
Comment (soerenstein) [reply src/components/CreateContact/CreateContact.js]: OnSubmit is a leftover from old code. Thanks for the hint!

File: src/App.js (hunk 2 above)
Comment (soerenstein) [reply src/App.js:18]: The router was part of the setup, so I already prepared it for upcoming components.

File: src/components/CreateContact/CreateContact.js (hunk 3 above)
Comment (soerenstein) [reply src/components/CreateContact/CreateContact.js]: I added the types to the inputs. Thanks!

**Pull request merged**